tests/pathfinding/test_rest.py

```diff
@@ -22,6 +22,58 @@
 ID_123 = 123
 
 
+#
+# tests for /_debug endpoint
+#
+
+
+def test_get_paths_via_debug_endpoint(
+    api_sut: ServiceApi, api_url: str, addresses: List[Address], token_network_model: TokenNetwork
+):
+    url = api_url + f'/{token_network_model.address}/paths'
+    url_debug = api_url + f'/_debug/routes/{token_network_model.address}/{addresses[0]}'
+    url_debug_incl_target = (
+        api_url + f'/_debug/routes/{token_network_model.address}/{addresses[0]}/{addresses[2]}'
+    )
+
+    data = {'from': addresses[0], 'to': addresses[2], 'value': 10, 'max_paths': DEFAULT_MAX_PATHS}
+    response = requests.post(url, json=data)
+    assert response.status_code == 200
+    paths = response.json()['result']
+    assert len(paths) == 1
+    assert paths == [{'path': [addresses[0], addresses[1], addresses[2]], 'estimated_fee': 0}]
+
+    # now there must be a debug endpoint for that specific route
+    response_debug = requests.get(url_debug)
+    assert response_debug.status_code == 200
+    request_count = response_debug.json()['request_count']
+    assert request_count == 1
+    responses = response_debug.json()['responses']
+    assert responses == [
+        {
+            'source': addresses[0],
+            'target': addresses[2],
+            'routes': [{'path': [addresses[0], addresses[1], addresses[2]], 'estimated_fee': 0}],
+        }
+    ]
+
+    # now there must be a debug endpoint for that specific route and that specific target
+    response_debug_incl_target = requests.get(url_debug_incl_target)
+    assert response_debug_incl_target.status_code == 200
+    request_count = response_debug_incl_target.json()['request_count']
+    assert request_count == 1
```

Comment:
I'd like to see that the `target_address`  is actually used to filter something out. Can you add the same test with a different `target_address`  below, so we see that `request_count`  becomes 0.

Reply:
done